docs/agile-methodologies.md

```diff
@@ -28,4 +28,107 @@ It consists of the elaboration of a table or diagram in which three columns of t
+8. Confirm any assumptions or dependencies discovered during planning and record.
+9. ScrumMaster calls for a group consensus on the plan.
+10. Team and Product Owner signal if this is the best plan they can make given what they know right now.
+11. Get back to work.
+
+
+### **Grooming Session**
+Backlog grooming, also referred to as backlog refinement or story time, is a recurring event for agile product development teams. The primary purpose of a backlog grooming session is to ensure the next few sprints worth of user stories in the product backlog are prepared for sprint planning. Regular backlog grooming sessions also help ensure the right stories are prioritized.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/UpEBfS9SZGM" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### Who is involved?
+
+- **Product Owner:** Is tasked with facilitating backlog refinement sessions. However, that doesn’t mean they are solely responsible for holding backlog grooming sessions.
+- **Team Members:** These events are meant to be collaborative. That means the entire cross-functional team should be represented at refinement sessions.
+- **QA representatives:** You need the combined expertise of the various functions on your team to effectively flesh out your user stories.
+
+### Grooming steps
+
+1. Removing user stories that no longer appear relevant.
+2. Creating new user stories in response to newly discovered needs.
+3. Re-assessing the relative priority of stories.
+4. Assigning estimates to stories which have yet to receive one
+correcting estimates in light of newly discovered information.
+5. Splitting user stories which are high priority.
+6. Get back to work.
+
+### **Retrospective Session** 
+
+Is a meeting held after a product ships to discuss what happened during the product development and release process, with the goal of improving things in the future based on those learnings and conversations.  
+An agile retrospective forces the entire team to pause and reflect on what transpired and discuss what worked and what didn’t during a particular project.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/MiaZhJyYUj0" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### How often should you hold retrospectives?
+
+Retrospectives can be held more frequently, including for minor releases, each sprint or even at daily or weekly standups.
+
+### **Demo Session**
+
+The sprint demo is invaluable for keeping stakeholders up to speed with the progress of product development. It allows them to feedback and discuss with the Product Owner and Scrum team any possible amendments to the Product Backlog which would help to maximise value.  
+He sprint demo shouldn’t take up too much of a Scrum team’s time. Ensuring that the sprint review meeting is an informal affair where questions, feedback and discussion are welcome – allows for prep time to be kept to a minimum.
```

Comment:
Is **H**e or **Th**e at the beginning of, I'm not sure:
> He sprint demo shouldn’t take up too much of a Scrum team’s time

Reply:
Is The, thanks!!